test/Faker/Provider/BaseTest.php

```diff
@@ -337,6 +337,7 @@ public function regexifyDataProvider()
             ['[a-z]{2,3}', 'brackets quantifiers on character class range'],
             ['(a|b){2,3}', 'brackets quantifiers on alternation'],
             ['\.\*\?\+', 'escaped characters'],
+            ['[.]', 'literal dot'],
```

Comment:
Maybe it's worth to add a test asserting that calling `regexify('[.]')` will always return `"."`?

Reply:
Yeah good idea, added it to the already existing test cases here:
https://github.com/FakerPHP/Faker/pull/135/files#diff-c648f7b997049e16add4fc6034d0c6f41f5c6b0c6a52a14b7fe4dd797a2a873cR313